runtime/drivers/duckdb/model_manager.go

```diff
@@ -10,8 +10,10 @@ import (
 )
 
 type ModelInputProperties struct {
-	SQL  string `mapstructure:"sql"`
-	Args []any  `mapstructure:"args"`
+	SQL         string `mapstructure:"sql"`
+	Args        []any  `mapstructure:"args"`
+	PreInitSQL  string `mapstructure:"pre_init_sql"`
+	PostInitSQL string `mapstructure:"post_init_sql"`
```

Comment:
"Pre init" sounds like something that runs *before* init, and "post init" sounds like something that runs *after* init (but still *before* the actual model).

I'll suggest one of these combinations instead:
- `pre_exec` and `post_exec` (advantage: "exec" conveys that it's a statement, not a SELECT like the normal `sql:`)
- `pre_sql` and `post_sql` (advantage: aligns closer with the existing `sql:` terminology, although it might be confusing since they're not SELECT statements)
- `init_sql` and `post_sql` (advantage: aligns with the `init_sql` terminology in the connector YAML)

Reply:
`pre_exec` and `post_exec` sounds good to me.